tests/UI/specs/OnlyRawDataNotification_spec.js

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Matomo - free/libre analytics platform
+ *
+ * Only Raw Data Notification screenshot tests.
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+describe("OnlyRawDataNotification", function () {
+    this.fixture = "Piwik\\Tests\\Fixtures\\OneVisit";
+
+    const generalParams = 'idSite=1&period=week&date=today';
```

Comment:
to ensure that might not fail randomly e.g. on monday around midnight it might be better to use a fix range like `period=range&date=2021-03-01,today`. As archiving is disabled it shouldn't matter that this range will contain more and more days...